### ISSUE TITLE
Maven central repo contains wrong sources and docs artifacts

### ISSUE THREAD

**Issue by nakedpony** [pusher/pusher-websocket-java#108]
The `pusher-java-client-1.1.2-sources.jar` contains docs and the `pusher-java-client-1.1.2-javadoc.jar` contains sources.

Please check the command and the output below. Could you please reupload the proper versions.

`curl -LO http://search.maven.org/remotecontent?filepath=com/pusher/pusher-java-client/1.1.2/pusher-java-client-1.1.2-sources.jar && jar -tf *.jar > 1.log`

`1.log` contents:

```
META-INF/
META-INF/MANIFEST.MF
allclasses-frame.html
allclasses-noframe.html
constant-values.html
deprecated-list.html
help-doc.html
index-all.html
index.html
overview-frame.html
overview-summary.html
overview-tree.html
package-list
script.js
serialized-form.html
stylesheet.css
com/
com/pusher/
com/pusher/client/
com/pusher/client/AuthorizationFailureException.html
com/pusher/client/Authorizer.html
com/pusher/client/package-frame.html
com/pusher/client/package-summary.html
com/pusher/client/package-tree.html
com/pusher/client/Pusher.html
com/pusher/client/PusherOptions.html
com/pusher/client/channel/
com/pusher/client/channel/Channel.html
com/pusher/client/channel/ChannelEventListener.html
com/pusher/client/channel/ChannelState.html
com/pusher/client/channel/package-frame.html
com/pusher/client/channel/package-summary.html
com/pusher/client/channel/package-tree.html
com/pusher/client/channel/PresenceChannel.html
com/pusher/client/channel/PresenceChannelEventListener.html
com/pusher/client/channel/PrivateChannel.html
com/pusher/client/channel/PrivateChannelEventListener.html
com/pusher/client/channel/SubscriptionEventListener.html
com/pusher/client/channel/User.html
com/pusher/client/connection/
com/pusher/client/connection/Connection.html
com/pusher/client/connection/ConnectionEventListener.html
com/pusher/client/connection/ConnectionState.html
com/pusher/client/connection/ConnectionStateChange.html
com/pusher/client/connection/package-frame.html
com/pusher/client/connection/package-summary.html
com/pusher/client/connection/package-tree.html
com/pusher/client/util/
com/pusher/client/util/Factory.html
com/pusher/client/util/HttpAuthorizer.html
com/pusher/client/util/package-frame.html
com/pusher/client/util/package-summary.html
com/pusher/client/util/package-tree.html
src-html/
src-html/com/
src-html/com/pusher/
src-html/com/pusher/client/
src-html/com/pusher/client/AuthorizationFailureException.html
src-html/com/pusher/client/Authorizer.html
src-html/com/pusher/client/Pusher.html
src-html/com/pusher/client/PusherOptions.html
src-html/com/pusher/client/channel/
src-html/com/pusher/client/channel/Channel.html
src-html/com/pusher/client/channel/ChannelEventListener.html
src-html/com/pusher/client/channel/ChannelState.html
src-html/com/pusher/client/channel/PresenceChannel.html
src-html/com/pusher/client/channel/PresenceChannelEventListener.html
src-html/com/pusher/client/channel/PrivateChannel.html
src-html/com/pusher/client/channel/PrivateChannelEventListener.html
src-html/com/pusher/client/channel/SubscriptionEventListener.html
src-html/com/pusher/client/channel/User.html
src-html/com/pusher/client/connection/
src-html/com/pusher/client/connection/Connection.html
src-html/com/pusher/client/connection/ConnectionEventListener.html
src-html/com/pusher/client/connection/ConnectionState.html
src-html/com/pusher/client/connection/ConnectionStateChange.html
src-html/com/pusher/client/util/
src-html/com/pusher/client/util/Factory.html
src-html/com/pusher/client/util/HttpAuthorizer.html
```

**Comment by cowwoc** [comment]
Can someone please fix this? It's very annoying to use your library in an IDE until this is fixed.

**Comment by zmarkan** [comment]
Thanks for flagging, I'll look into it.

**Comment by zmarkan** [comment]
I've fixed this in 1.6.1 - it should take a while until it propagates on Maven Central in a few hours. https://github.com/pusher/pusher-websocket-java/pull/167

**Comment by cowwoc** [comment]
Thank you
